packages/api/src/external/carequality/process-outbound-patient-discovery-resps.ts

```diff
@@ -62,11 +70,53 @@ export async function processOutboundPatientDiscoveryResps({
       },
     });
 
-    log(`Completed.`);
+    const rerunPdOnNewDemographics = cqData?.rerunPdOnNewDemographics;
+    const facilityId = cqData?.discoveryFacilityId;
+    let foundNewDemographics = false;
+    if (rerunPdOnNewDemographics) {
+      foundNewDemographics = checkForNewDemographics(patient, cqLinks);
+    }
+
+    let scheduledPdRequest = cqData?.scheduledPdRequest;
+    if (foundNewDemographics && rerunPdOnNewDemographics && facilityId) {
+      const updatedPatient = await scheduleOrRunPatientDiscovery({
+        patient,
+        facilityId,
+        rerunPdOnNewDemographics: false,
```

Comment:
so just to check, because this is set to false, we only will do one round of updates and cant end up in an infinite loop of updating right?

Reply:
Yes.